spec/takeaway_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'takeaway'
+
+describe Takeaway do
+  it 'Is an instance of the takeaway class' do
+    expect(subject).to be_an_instance_of Takeaway
+  end
```

Comment:
you can consider to use a describe statement for your initialize method as you did for the other methods